addons/web_editor/static/src/js/wysiwyg/wysiwyg_iframe.js

```diff
@@ -215,10 +215,6 @@ function getWysiwygIframeContent(params) {
         ${assets.jsContents.join('\n')}
 
         <script type="text/javascript">
-            odoo.define('web.session', [], function () {
-                return window.top.odoo.__DEBUG__.services['web.session'];
-            });
```

Comment:
This one I don't know what it could break but it would surprise me if removing it without replacing it by something else wouldn't break something somewhere as well. Though this one was only moved by @Goaman so it was already there so it's possible it's there for historical reasons and is not needed anymore, but it's impossible to tell from the diff here.

Reply:
I imagine it was there to not bind session again (session.is_bound = session.session_bind()) which reload the translations/localization. These are loaded by services now.
If it's not for this then I don't know the reason.

Reply:
I wouldn't be worried to remove it if it does not break any test for that one.